### DISCUSSION TITLE
Ability to view discussions started by you or where you're participated

### DISCUSSION THREAD

**Discussion by EricKotato** [community/community#2880] [Discussions] [answered]
Site header has "Pull requests" and "Issues" links, where you can search and filter issues like this:

![](https://user-images.githubusercontent.com/2903496/101588569-c1704d80-39f7-11eb-962d-0a8f2bf6c735.png)

However, there is no such thing for discussions, though it would be useful.

**Comment by EricKotato** [comment, marked as answer]
Nevermind, found it in profile menu.

![](https://user-images.githubusercontent.com/2903496/101590334-427d1400-39fb-11eb-971c-9d93e758c559.png)

**Reply by michael-hawker** [reply]
That view only shows the discussions you started though, not all the ones you've participated. If you're already in a discussion forum, it'd still be nice to filter to the list of things you've upvoted, commented on, or started. Those are all important facets.

**Reply by joel-ekstep** [reply]
It looks like a couple of these options are available within the 'Your discussions' section. I can see a 'Created' and 'Commented' tab in the section.
